extensions/amp-video/0.1/amp-video.js

```diff
@@ -352,7 +352,7 @@ class AmpVideo extends AMP.BaseElement {
         }
         throw reason;
       })
-      .then(() => this.onVideoLoaded_());
+      .then(() => (this.isManagedByPool_() ? null : this.onVideoLoaded_()));
```

Comment:
We shouldn't trigger onVideoLoaded if it's managed by the pool because the pool will send this event by itself on `resetOnDomChange`